tests/functional/pom.xml

```diff
@@ -104,6 +104,37 @@
         </dependency>
 
         <dependency>
+            <groupId>org.eclipse.aether</groupId>
```

Comment:
Those dependencies were required before this PR but were included in the classpath because another dependency, that does not depend on this anymore, was using them.